### DISCUSSION TITLE
distance calculation issue

### DISCUSSION THREAD

**Discussion by zzploveyou** [Turbo87/utm#74] [Q&A]
point a
`utm.from_latlon(2,120)`
`(166223,221366,51,'N')`

point b
`utm.from_latlon(1,120)`
`(166072,110682,51,'N')`

point c
`utm.from_latlon(-1,120)`
`(166072,9889317,51,'M')`

9889317 >> 110682

For point a and point b, We can calculate distance using the formula sqrt((x1-x2)**2+(y1-y2)**2). But how can i calculate the distance between point b and c since they have different zone_letters?

Is there any way that can converts latlons in the same coordinate / scale?

**Comment by sdhiscocks** [comment]
Currently this isn't possible, as you are unable to force points below the equator to be in a common zone with those above the equator. Proposed fix in #46

**Comment by bartvanandel** [comment]
That fix has been merged, so you can now apply your logic by forcing a zone.

Note that taking the Euclidian distance won't provide the most accurate results, because they operate on a flat plane, and UTM coordinates are approximately spherical, but that's outside the scope of this library.

Also, further away from the central point of each zone, coordinates become less accurate. So unless your points are relatively close together, your computed distances will be quite inaccurate.
